From 9978746e91b8c6789d792bdc926306db0873c74c Mon Sep 17 00:00:00 2001
From: Dodji Seketeli <dodji@redhat.com>
Date: Sat, 27 Feb 2010 22:14:38 +0100
Subject: [PATCH] Fix build with gcc 4.5

	* src/common/nmv-object.cc: Class name is Object, not
	Object::Object.
---
 src/common/nmv-object.cc |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/common/nmv-object.cc b/src/common/nmv-object.cc
index fc1f0d1..679b8ae 100644
--- a/src/common/nmv-object.cc
+++ b/src/common/nmv-object.cc
@@ -59,7 +59,7 @@ Object::Object (Object const &a_object):
     *m_priv = *a_object.m_priv;
 }
 
-Object::Object&
+Object&
 Object::operator= (Object const &a_object)
 {
     if (this == &a_object)
-- 
1.7.2.3

